tests/serverpod_test_server/lib/src/endpoints/reflection.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:serverpod/serverpod.dart';
+
+/// This class is meant for reflecting the received headers, auth keys in endpoint invocations.
+class ReflectionEndpoint extends Endpoint {
```

Comment:
Nit: Since reflection is a common concept in programming (although not supported by Dart), maybe we could name this something more explicit like `ReflectHttpHeadersEndpoint` to avoid any confusion. Maybe a bit of stretch, but I think these small things add up and make the code base easier to navigate.

Reply:
I propose to use Alex' suggestion RequestReflectionEndpoint. The intent is not to have only HTTP specific things but generally for endpoint methods.

Reply:
I still think the use of `Reflection` like this is ambiguous and I would avoid it completely. May I suggest `ReflectRequestEndpoint` or perhaps `EchoRequestEndpoint` , which is more unambiguous? (cc @SandPod since you also commented on this)

Reply:
`Echo` seems like a good suggestion in that case? Maybe even `EchoRequestAttributeEndpoint`?
Good point about mixing this up with general reflections in programming.